cfgov/jinja2/v1/_includes/organisms/form.html

```diff
@@ -0,0 +1,67 @@
+{# ==========================================================================
+
+   common_form.form()
+
+   ==========================================================================
+
+   Description: Returns a date picker.
+
+   action:       The action for the form. Default is '/'.
+
+   classes:      A comma separated string list to be added to the form.
+
+   fields:       A list of forms fields to be rendered within the form.
+
+   method:       The method for the form. Default is 'GET'.
+
+   submit_label: The action for the form submit button. Default is '/'.
+
+   ========================================================================== #}
+{% set options = {
+    'action':       '/',
+    'classes':      '',
+    'errors' :       None,
+    'method':       'GET',
+    'submit_label': 'Submit',
+    'csrf_token':   '{{ csrf_token }}'
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/65fd027c19ba7c5096d15a32912caee61807f7c4